src/lib/Stepper.svelte

```diff
@@ -1,27 +1,62 @@
 <script lang="ts">
 	import { createEventDispatcher } from 'svelte';
-
 	export type Step = $$Generic<{ label: string }>;
+	import type { SvelteHTMLElements } from 'svelte/elements';
+	type $$Props = SvelteHTMLElements['section'];
+	import { twMerge } from 'tailwind-merge';
+
+	let className: $$Props['class'] = undefined;
+	export { className as class };
+
 	export let steps: Step[];
+	let activeStep: Step;
```

Comment:
I think we would need to export this so it can be set by the consumer
```suggestion
	export let activeStep: Step | undefined = undefined;
```

then only set the activeStep below if its not already defined